pontoon/base/models.py

```diff
@@ -94,7 +95,14 @@ def user_display_name(self):
     name = self.first_name or self.email.split('@')[0]
     return u'{name} <{email}>'.format(name=name, email=self.email)
 
+
+@property
+def can_approve_translations(self):
+    return self.groups.filter(permissions__codename='can_translate_locale').exists()
```

Comment:
Is it possible to use `has_perm` here? I wonder if it returns true if you have the given permission for a single project?